src/http_upload/mod_http_upload.erl

```diff
@@ -122,6 +123,36 @@ get_urls(Host, Filename, Size, ContentType, Timeout) ->
     mod_http_upload_backend:create_slot(UTCDateTime, Token, Filename,
                                         ContentType, Size, NewOpts).
 
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+        items = #{<<"iqdisc">> => mongoose_config_spec:iqdisc(),
+                  <<"host">> => #option{type = string,
+                                        validate = domain_template},
+                  <<"backend">> => #option{type = atom,
+                                           validate = {module, mod_http_upload}},
+                  <<"expiration_time">> => #option{type = integer,
+                                                   validate = positive},
+                  <<"token_bytes">> => #option{type = integer,
+                                               validate = positive},
+                  <<"max_file_size">> => #option{type = integer,
+                                                 validate = positive},
+                  <<"s3">> => s3_spec()
+        }
+    }.
+
+s3_spec() ->
+    #section{
+        items = #{<<"bucket_url">> => #option{type = string,
+                                              validate = url},
+                  <<"add_acl">> => #option{type = boolean},
+                  <<"region">> => #option{type = string},
+                  <<"access_key_id">> => #option{type = string},
+                  <<"secret_access_key">> => #option{type = string}
```

Comment:
These all should be mandatory - see `mod_http_upload_s3.erl`. I think we should change the docs and the implementation (and tests) to reflect that.